docs/deploy_daos_cluster_example.md

```diff
@@ -163,14 +136,14 @@ gcloud compute instance-groups managed wait-until 'daos-server' \
 
 ## Perform DAOS administration tasks
 
-After your DAOS cluster has been deployed you can log into the first DAOS client instance to perform administrative tasks.
+After your DAOS cluster has been deployed you can log into the first DAOS server instance to perform administrative tasks.
```

Comment:
In DAOS v2.2 the dmg command is no longer in the daos-client package. The daos-client is no longer used as an admin node. In the future we need to add a daos_admin terraform module and use that to provision an admin instance as part of the DAOS cluster. For now any `dmg` commands must be run as root on the first DAOS server instance. 

If, preferred, I can change the packer configuration to install the daos-admin package in the client images so that the instructions for using dmg do not need to change.